tests/cpp_api_tests/rocAL_audio_unittests/rocAL_audio_unittests.cpp

```diff
@@ -163,6 +178,13 @@ int test(int test_case, const char *path, int qa_mode, int downmix, int gpu) {
             RocalFloatParam p_preemph_coeff = rocalCreateFloatParameter(0.97);
             rocalPreEmphasisFilter(handle, decoded_output, true, p_preemph_coeff, preemph_border_type, tensorOutputType);
 
+        } break;
+        case 2: {
+            std::cout << ">>>>>>> Running SPECTROGRAM" << std::endl;
+            case_name = "spectrogram";
+            std::vector<float> window_fn;
+            rocalSpectrogram(handle, decoded_output, true, window_fn, true, true, RocalSpectrogramLayout::ROCAL_FT, 2, 512, 320, 160, ROCAL_FP32);
```

Comment:
Why an empty float vector is passed for window_fn @HazarathKumarM

Is this intended @swetha097